stracker/items/models.py

```diff
@@ -0,0 +1,76 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from core.models import BaseAbstractModel
+from customers.models import Customer
+
+
+class Category(BaseAbstractModel):
+    """
+    Model to hold item categories (types)
+    """
+
+    name = models.CharField(max_length=255, verbose_name=_("Category Name"))
+
+    class Meta:
+        verbose_name = _("category")
+        verbose_name_plural = _("categories")
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Fridge(BaseAbstractModel):
+    """
+    Model to hold storage locations
+    """
+    name = models.CharField(verbose_name=_("name"), max_length=128, unique=True)
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Shelf(BaseAbstractModel):
+    """
+    Model to hold storage divisions
+    """
+    name = models.CharField(verbose_name=_("name"), max_length=128, unique=True)
+    fridge = models.ForeignKey(Fridge, verbose_name=_("Fridge"), on_delete=models.CASCADE)
+    owner = models.ForeignKey(Customer,verbose_name=_("Owner"), on_delete=models.CASCADE)
+
```

Comment:
is_active alanı ile kullanım dışı olup olmadığı kontrol edilebilir. db'de bu alan var.

Reply:
Shelf'in aktif olmadığı bir durum hayal etmekte zorlandığım için eklemedim. Ancak belki fridge'e eklemek doğru olabilir. 

Yani buzdolabında tek bir rafın iptal edilmesi yerine bütün dolabın iptal edilmesi gibi.